### PR TITLE
Update github workflows

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose

Fixes:

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: .github/workflows/central-publish.yml [hunk 1]
```diff
@@ -84,4 +98,4 @@ jobs:
           REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
         run: |
           sed -i 's/version=\(.*\)-SNAPSHOT/version=\1/g' gradle.properties
-          ./gradlew clean build -PpublishToCentral=true
\ No newline at end of file
+          ./gradlew clean build -PpublishToCentral=true
```
Comment (warunalakshitha) [inline .github/workflows/central-publish.yml:101]: shall we add new line

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -11,38 +11,60 @@ jobs:
...
+        run: |
+          echo '${{ toJson(secrets) }}' | jq -r 'to_entries[] | .key + "=" + .value' | Out-File -FilePath $env:GITHUB_ENV -Append 
+
       - name: Build with Gradle
         env:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.GITHUB_TOKEN }}
+          JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8
         run: ./gradlew.bat build -x test
         # Disabling tests because no docker in git-action windows
```
Comment (warunalakshitha) [inline .github/workflows/pull-request.yml]: Need new line

File: .github/workflows/trivy-scan.yml [hunk 3]
```diff
@@ -3,32 +3,12 @@ name: Trivy
...
-          scan-type: 'rootfs'
-          scan-ref: '/github/workspace/ballerina/lib'
-          format: 'table'
-          timeout: '10m0s'
-          exit-code: '1'
+  call_workflow:
+    name: Run Trivy Scan Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-library/.github/workflows/trivy-scan-template.yml@main
+    secrets: inherit
```
Comment (warunalakshitha) [inline .github/workflows/trivy-scan.yml]: New Line missing

File: .github/workflows/build-with-bal-test-graalvm.yml [hunk 4]
```diff
@@ -37,4 +37,3 @@ jobs:
       native_image_options: '-J-Xmx7G ${{ inputs.native_image_options }}'
       additional_windows_build_flags: '-x test'
     secrets: inherit
```
Comment (daneshk) [inline .github/workflows/build-with-bal-test-graalvm.yml:39]:
```suggestion
    secrets: inherit
    
```

File: .github/workflows/build-timestamped-master.yml [hunk 5]
```diff
@@ -62,4 +75,3 @@ jobs:
           nexusPassword: ${{ secrets.NEXUS_PASSWORD }}
         run: |
           ./gradlew clean publishMavenJavaPublicationToWSO2NexusRepository -x test --scan --no-daemon
```
Comment (daneshk) [inline .github/workflows/build-timestamped-master.yml:77]:
```suggestion
          ./gradlew clean publishMavenJavaPublicationToWSO2NexusRepository -x test --scan --no-daemon
          
```

**Review by warunalakshitha — APPROVED** [review]

**Pull request merged**